package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ember/string",
-  "version": "0.0.0",
+  "version": "1.0.0",
```

Comment:
I generally only update this during publishing. Nothing wrong with doing it this way though...